doc/release.md

```diff
@@ -0,0 +1,78 @@
+# Release
+
+**JVS-PLUGIN-GITHUB is not an official Google product.**
+
+We leverage [goreleaser](https://goreleaser.com/) for SCM (GitHub) and image
+release.
+
+## New Release
+
+- If a new JVS base image is needed, update [Dockerfile](https://github.com/abcxyz/jvs-plugin-github/blob/main/Dockerfile)
+to point to a new JVS base image, and merge the PR.
+
+- Send a PR to update all dependencies For Go
+```sh
+go get -u && go mod tidy
+```
+
+- Create a tag using `.github/workflows/create-tag.yml` on default branch and
+     run the workflow with below inputs.
+
+  - tag name with format `v0.x.x`, using semantic versioning.
```

Comment:
Maybe `v0` should be changed. We won't be on major version 0 forever.

Reply:
changed to v1.x.x